example/behat.yml

```diff
@@ -48,6 +48,12 @@ default:
       print_outp: true
       loop_break: true
 
+     # Bex Screenshot
+    Bex\Behat\ScreenshotExtension:
+      image_drivers:
+        local:
+          screenshot_directory: "%paths.base%/<change_screenshots_folder_name>/"
```

Comment:
Can we assume a default folder name, which works out of the box (is created automatically)?

Reply:
If you don't provide a folder screenshots are stored in `/tmp/behat-screenshot/`

![image](https://user-images.githubusercontent.com/5762835/58196386-db8df200-7cd2-11e9-9ee3-2faa27365ed1.png)

Reply:
If it's just set to `%paths.base%/screenshots/` by default, where will the screenshots go?

Reply:
`%paths.base%` refers to the base of behat. This is the folder which contains the `behat.yml` and the `features` folder. In other words the `screenshots` folder will be created on the same level as the `features` folder.

Reply:
> In other words the screenshots folder will be sibling of the `features` folder.
That sounds like a fine place to place the `screenshots` folder, so let's just go with that. This way users won't have to edit anything out of the box.

Reply:
I agree. Changes to `behat.yml` are commited.